import-export-cli/docs/apictl.md

```diff
@@ -22,6 +22,7 @@ apictl [flags]
 ### SEE ALSO
 
 * [apictl add](apictl_add.md)	 - Add Environment to Config file
+* [apictl bundle](apictl_bundle.md)	 - Archive any project to zip format
```

Comment:
Shall we change this command description as follows to provide a more descriptive one?

```suggestion
* [apictl bundle](apictl_bundle.md)	 - Archive any Source project artifact to zip format
```

Reply:
Fixed with 9a12cbd